sellotape/sellotape_dj/main_app/models.py

```diff
@@ -1,8 +1,32 @@
 from django.db import models
 from django.contrib.auth.models import User
 
+# https://docs.djangoproject.com/en/dev/topics/auth/customizing/#extending-the-existing-user-model
+class Profile(models.Model):
```

Comment:
Please add 'youtube_channel_link' and 'twitch_channel_link' fields.

Reply:
Great note!
Done ;-)